src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Logging/ILogger.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Razor.Logging;
+
+internal interface ILogger
```

Comment:
Consider either renaming this to `IRazorLogger` to match the other interfaces here, or renaming the other interfaces to remove the "Razor" bit.